### PR TITLE
fix: Refactor toggle solution in store vue

### LINKED ISSUE #1111: Refactor toggle solution in store vue file

### PR DESCRIPTION
**Pull request by talwinder50**
closes #1111

Signed-off-by: talwinder50 <talwinderkaur50@gmail.com>

### CODE REVIEW

File: cmd/wallet-web/src/pages/Store.vue [hunk 1]
```diff
@@ -167,13 +170,24 @@ export default {
...
+        this.processedCredentials.push({ ...credential, showDetails: true });
+      } else {
+        data.verifiableCredential.map((vc) => {
+          const manifest = this.getManifest(vc);
+          const credential = this.getCredentialDisplayData(vc, manifest);
+          this.processedCredentials.push({ ...credential, showDetails: false });
+        });
+      }
+
+      console.log('processedCredentials', JSON.stringify(this.processedCredentials, null, 2));
```
Comment (sudeshrshetty) [inline cmd/wallet-web/src/pages/Store.vue:190]: left over?

**Review by sudeshrshetty — APPROVED** [review]

**Review by rolsonquadras — APPROVED** [review]

File: cmd/wallet-web/src/pages/Store.vue (hunk 1 above)
Comment (talwinder50) [reply cmd/wallet-web/src/pages/Store.vue:190]: No it was there before as well but i can delete

Comment (birtony) [reply cmd/wallet-web/src/pages/Store.vue:190]: Let's delete the console

**Review by birtony — APPROVED** [review]
LGTM

**Pull request merged**